api/graphql/schema.py

```diff
@@ -537,6 +537,8 @@ class GraphQLSequencingGroup:
 
     internal_id: strawberry.Private[int]
     sample_id: strawberry.Private[int]
+    created_on: datetime.date
+    assay_meta: list[strawberry.scalars.JSON]
```

Comment:
I did have that same thought after I had implemented `assay_meta`, however the issue is that all assay meta will be returned even if they don't match the search filters. This is not necessarily a bad thing, but it might be confusing to a user if they also see assay meta included that doesn't match their search criteria.

Reply:
That's why querying on a nested subtype is a little awkward, because you implicitly define a "matching one of" relationship. It's subtle but I actually think that's preferable for this - because you want to align all relevant assays as part of a sequencing group - regardless of which batch the experiment is in. (You don't want to align a partial set of assays on an sg), and in fact, maybe we should remove the filters on sg -> assay to not really allow people to accidentally do that.

Reply:
I removed `assay_meta` in favour of returning the nested `assays.meta` field